fees/helpers.py

```diff
@@ -16,3 +17,10 @@ def get_purchaser_model():
         raise ImproperlyConfigured(
             "FEES_PURCHASER_MODEL refers to model '%s' that has not been installed" % fees_settings.PURCHASER_MODEL
         )
+
+def invalidate_purchaser_cache(cache_version):
```

Comment:
Maybe better approach would be:

```
def invalidate_purchaser_cache(purchaser)
    cache_version = purchaser.cache_version
```

and usage:

``invalidate_purchaser_cache(self.purchaser)``